packages/twenty-front/src/pages/onboarding/InviteTeam.tsx

```diff
@@ -218,6 +223,9 @@ export const InviteTeam = () => {
           fullWidth
         />
       </StyledButtonContainer>
+      <StyledActionSkipLinkContainer>
+        <ActionLink onClick={handleSubmit(onSubmit)}>Skip</ActionLink>
```

Comment:
think you can add a `skip` parameter to make sure no invitation mail is send when clicking on skip